manifests/plugins/signalfx.pp

```diff
@@ -22,11 +21,11 @@
   collectd::check_and_install_package { 'signalfx-collectd-plugin':
     before  => File['load Signalfx plugin']
   }
-  if $::osfamily == 'Redhat' {
```

Comment:
Added the if block back. Have seen the deb packaging builder scripts available in a separate repo, are the rpm build scripts also available?